omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -371,12 +371,13 @@
 default['private_chef']['nginx']['ssl_certificate'] = nil
 default['private_chef']['nginx']['ssl_certificate_key'] = nil
 default['private_chef']['nginx']['ssl_dhparam'] = nil
-default['private_chef']['nginx']['ssl_country_name'] = "US"
-default['private_chef']['nginx']['ssl_state_name'] = "WA"
```

Comment:
I doubt many users were setting these in their configuration, but what will these default to now?